app/src/main/java/org/mozilla/fenix/components/appstate/AppAction.kt

```diff
@@ -6,10 +6,14 @@ package org.mozilla.fenix.components.appstate
 
 import mozilla.components.lib.state.Action
 import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.wallpapers.Wallpaper
 
 /**
  * [Action] implementation related to [AppStore].
  */
 sealed class AppAction : Action {
+    object InitAction : AppAction()
     data class UpdateInactiveExpanded(val expanded: Boolean) : AppAction()
+    object SwitchToNextWallpaper : AppAction()
```

Comment:
🚗 Future consideration: as we add more actions in the future, we should organize these actions to have some categorization of actions similar to BrowserAction (e.g. TabListAction)

Reply:
Same goes for the data classes in AppState.

Neither of these need to be actioned on for now. 🙂